### ISSUE TITLE
Ide specific key conf is ignored for everything but "debug"

### ISSUE THREAD

**Issue by Ketec** [mac-cain13/xdebug-helper-for-chrome#95]
I have it set to phpstorm -  but when i switch to trace/profile - XDEBUG_ECLIPSE key is sent instead.

Had to use the other  two custom options to force phpstorm key.

**Comment by gwharton** [comment]
How did you override it with custom options?

**Comment by Ketec** [comment]
In the extension options  Trace Trigger Value and Profile Trigger Value

**Comment by gwharton** [comment]
Yes, sorry. Just inspected the code and saw that! Its an easy workaround once you know!
